ci/infra/testrunner/tests/test_cilium.py

```diff
@@ -52,3 +52,34 @@ def test_cillium(deployment, kubectl):
         time.sleep(30)
 
     assert all_reachable
+
+
+def test_cilium_l7(deployment, kubectl):
+    """
+    GIVEN cilium is properly configured
+    AND an L7 policy that restricts access to an endpoint is applied
+    WHEN a unautorized request is made to the endpoint
+    THEN Access is denied
+    """
+    landing_req = 'curl -sm10 -XPOST deathstar.default.svc.cluster.local/v1/request-landing'
+    exhaust_port = 'curl -s -XPUT deathstar.default.svc.cluster.local/v1/exhaust-port'
+
+    logger.info("Deploy deathstar")
+    kubectl.run_kubectl("create -f https://raw.githubusercontent.com/cilium/cilium/v1.5/examples/minikube/http-sw-app.yaml")
```

Comment:
should we have those external yamls locally? Seems prone to failure (both network and repo gone wrong) plus it requires and extra step to check what is actually being deployed instead of being in the same repo at a glance

Reply:
🤔 we could. It was originally done this way. I'll poke around as to how hard that would be to do in our test infra? I would think not hard?
The only downside of this is we now have to manage the file, but I don't think that should be that bad.

Reply:
not a blocker, its just an improvement that can be done down the line